recipe/meta.yaml

```diff
@@ -33,12 +32,18 @@ build:
 
     {{ llama_args("NATIVE", "OFF") }}        # [(osx and arm64) or win]
     {{ llama_args("ACCELERATE", "ON") }}     # [osx]
-    {{ llama_args("METAL", "ON") }}          # [osx]
+    {{ llama_args("METAL", "ON") }}          # [osx and arm64]
+    {{ llama_args("METAL", "OFF") }}         # [osx and x86_64]
     {{ llama_args("CUBLAS", "ON") }}         # [cuda_compiler_version != "None"]
 
     {{ llama_args("BLAS", "ON") }}                        # [not osx and cuda_compiler_version == "None"]
     {{ llama_args("BLAS_VENDOR", "Intel10_64_dyn") }}     # [blas_impl == "mkl"]
 
+    - export CMAKE_GENERATOR=Ninja    # [unix]
+    - set CMAKE_GENERATOR=Ninja       # [win]
+    - set CMAKE_GENERATOR_PLATFORM=   # [win]
+    - set CMAKE_GENERATOR_TOOLSET=    # [win]
```

Comment:
@YYYasin19, it looks like CUDA is only configured correctly if we use Ninja, the VS Generator does not work!